server/service/osquery_utils/queries.go

```diff
@@ -1123,7 +1127,11 @@ var softwarePythonPackagesWithUsersDir = DetailQuery{
 		FROM cached_users CROSS JOIN python_packages USING (uid)
 	`),
 	Platforms: append(fleet.HostLinuxOSs, "darwin", "windows"),
-	Discovery: `SELECT 1 FROM osquery_info WHERE version_compare(version, '5.16.0') >= 0`,
+	Discovery: `SELECT 1 FROM (
```

Comment:
Great catch!

Let's add a comment for future archeologists to know why we are not using `version_compare`.

And we should probably have a discussion around the minimum supported version of osquery (5.8.2 is two years old).

Reply:
Good enough to put that in a new PR to `main`? FWIW I figured the blame would lead back to this PR, which has context.

Reply:
For sure on `main`. Also no rush.